CMD/CMD.csproj

```diff
@@ -8,7 +8,7 @@
 
   <ItemGroup>
     <PackageReference Include="FluentCommandLineParser" Version="1.4.3" />
-    <PackageReference Include="mzLib" Version="1.0.376" />
+    <PackageReference Include="mzLib" Version="0.0.27" />
```

Comment:
what's up with the mzlib version

Reply:
This is a local mzLib version. Once I get the changes into mzLib, this'll get updated to the most recent version.